specification/application_layer/functions.tex

```diff
@@ -86,9 +86,10 @@ \subsection{Network-wide time synchronization}
 The detailed specification of the time synchronization algorithm is provided in the documentation
 for the message type \DSDLReference{uavcan.time.Synchronization}.
 
-An important service type \DSDLReference{uavcan.time.GetSynchronizationMasterInfo}
+An important service type \DSDLReference{uavcan.time.GetSynchronizationMasterInfo} 
 is designed to provide nodes with information about the currently used time system
 and related data such as the number of leap seconds.
+% Would be better restated in active voice "\DSDLReference{uavcan.time.GetSynchronizationMasterInfo} is an important service type designed to provide nodes with information about the currently used time system and related data such as the number of leap seconds."
```

Comment:
Sure. Can we please incorporate this change here? Also, I see that the original phrase "an important service type" has no semantic weight and should be removed.

Reply:
> system time

It's supposed to be ["time system"](https://en.wikipedia.org/wiki/Time#Time_conversions), not "system time". As in, an approach to timekeeping.

Reply:
Thanks for the clarification. Just flipping time and system around: 

Use uavcan.time.GetSynchronizationMasterInfo to provide nodes with information about the currently used time system and related data like the number of leap seconds added.

Reply:
Good. There's one last thing (I suppose I should mention this in the readme): some time ago Scott suggested that we avoid addressing the reader directly for reasons of consistency, so we probably shouldn't use the imperative mood like "use the data type".

Reply:
how about just "uavcan.time.GetSynchronizationMasterInfo provides nodes with information about the currently used time system and related data like the number of leap seconds added." ?